app/views/posts/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1> Implement all posts for associated to a specific user</h1>
+  <%= render partial: 'partials/user', locals: {user: @user} %>
+   <% @user.posts.each do |post| %>
+     <%= render partial: 'partials/post', locals: { post: post } %>
+     <div class="all-comments">
+    <% post.comments.limit(5).each do |comment| %>
```

Comment:
- [ ] Please consider using the method you created earlier: 🤓 🔧 

https://github.com/ProgramKingAbel/tubongeApp/blob/eb0868d9b78f13de205bfbab241cd64ec1b4b71d/app/models/post.rb#L16-L18